src/element/collision.ts

```diff
@@ -207,6 +240,21 @@ const hitTestPointAgainstElement = (args: HitTestArgs): boolean => {
         "This should not happen, we need to investigate why it does.",
       );
       return false;
+    case "frame": {
+      // check top distance
+      const labelDiv = document.getElementById(
+        args.element.id,
+      ) as HTMLDivElement;
+      const disToElement = distanceToBindableElement(args.element, args.point);
+      const disToLabel =
+        labelDiv && args.appState
+          ? distanceToLabel(labelDiv, args.element, args.point, args.appState)
+          : Infinity;
+      return (
+        args.check(disToLabel, args.threshold) ||
+        args.check(disToElement, args.threshold)
+      );
+    }
```

Comment:
we can't be running DOM queries (or using DOM in any way shape or form) during hit tests.

What is hit testing against the frame name DOM needed for? If it is, we need to cache the DOM scene coords somewhere to be used in colission testing, but DOM def can't be part of these pure checks.